lib/biokbase/log.py

```diff
@@ -0,0 +1,411 @@
+"""
```

Comment:
standard kbase app module - adding it here as I plan to update the base image in the next PR